ceph/rados/monitor_workflows.py

```diff
@@ -281,3 +282,107 @@ def set_tiebreaker_mon(self, host) -> bool:
+        mon_quorum = mon_election_obj.get_mon_quorum()
+
+        for host in hosts:
+
+            if not rados_obj.check_daemon_exists_on_host(
+                host=host.hostname, daemon_type="mon"
+            ):
+                continue
+
+            mon_status, mon_status_desc = rados_obj.get_daemon_status(
+                daemon_type="mon", daemon_id=host.hostname
+            )
+
+            if mon_status != 1 or mon_status_desc != "running":
+                continue
+
+            cmd = (
+                f"cephadm shell ceph daemon mon.{host.hostname} connection scores dump"
+            )
+            out, err = host.exec_command(sudo=True, cmd=cmd)
+            mon_conn_score = json.loads(out)
+            log.info(
+                f"Connectivity score of all daemons in the cluster: \n {mon_conn_score}"
+            )
+
+            reports = mon_conn_score["reports"]
+
+            # Compare number of monitors from connection scores dump and
+            # ceph mon dump commands
+            if len(reports) != num_mons:
+                log.error("Invalid number of mons in connection scores dump")
+                return False
+
+            for report in reports:
+                current_mon_rank = report["rank"]
+
+                # collect peer mon ranks from ceph mon dump command
+                peer_mons_rank = [
+                    peer_mon_rank
+                    for peer_mon_rank in mon_quorum.values()
+                    if peer_mon_rank != current_mon_rank
+                ]
+                peer_scores = report["peer_scores"]
+
+                log.info(f"current mon: {current_mon_rank}")
+                log.info(f"peer mons: {peer_mons_rank}")
+
+                # Compare number of peer monitors in connection scores dump and
+                # ceph mon dump command
+                if len(peer_scores) != len(peer_mons_rank):
+                    log.error("Invalid number of peers in connection scores dump")
+                    return False
+
+                for peer_score_detail in peer_scores:
+                    log.info(
+                        f"peer mon rank: \
+                        {peer_score_detail['peer_rank']} \
+                        peer mon score: \
+                        {peer_score_detail['peer_score']}"
+                    )
+                    if peer_score_detail["peer_rank"] not in peer_mons_rank:
+                        log.error("peer mon score missing in connection scores dump")
+                        return False
+
+                    # Connection scores for monitors are valid if they are in
+                    # the range of 0 to 1
+                    if (
+                        peer_score_detail["peer_score"] < 0
+                        and peer_score_detail["peer_score"] > 1
+                    ):
+                        log.error("Invalid peer_score value in connection scores dump")
+                        return False
+
```

Comment:
Let's add info statement, something like "verified connection scores on mon / or on host  X, and all scenarios passed" before returning true